client/js/_author/components/assessments/question_types/question_common/settings.jsx

```diff
@@ -29,8 +29,8 @@ export default function questionSettings(props) {
             onBlur={props.updateItem}
           >
             <option value="639-2%3AENG%40ISO">English</option>
-            <option value="">French</option>
-            <option value="">Spanish</option>
+            <option value="">Hindi</option>
```

Comment:
You should fins what the real values are, and we may still want the other languages